coverage/generate-index-html.py

```diff
@@ -10,35 +10,67 @@
+    <meta name="robots" content="index, follow">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Node.js Core Coverage</title>
-    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" type="text/css">
-    <link rel="stylesheet" href="https://code.getmdl.io/1.1.3/material.yellow-indigo.min.css" />
-    <style media="screen" type="text/css">
-      .table-container {
-        display: flex;
-        justify-content: center;
-        margin-top: 50px;
-      }
+
+    <link rel="apple-touch-icon" href="https://nodejs.org/static/apple-touch-icon.png">
+    <link rel="icon" sizes="32x32" type="image/png" href="https://nodejs.org/static/favicon.png">
+
+    <link rel="stylesheet" href="https://nodejs.org/en/styles.css" media="all">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600">
+    <style>
+        #logo { margin-bottom: 1rem; }
+        #graphs {
+            overflow: hidden;
+            clear: both;
+            list-style-type: none;
+            margin: 0;
+            padding: 0;
+        }
+        #graphs li {
+            float: left;
+            border: 1px solid #999;
+            margin-left: 1%;
+            margin-bottom: 1rem;
+            padding: 0.25rem;
+            max-width: 48%;
+        }
+        #graphs li:nth-child(odd) {
+            clear: both;
+            margin-left: 0;
+        }
+        #graphs img {
+            max-width: 100%;
+        }
+        @media (max-width:850px) {
+            #graphs li {
+                float: none;
+                margin-left: 0;
+                max-width: 100%;
+            }
+        }
     </style>
   </head>
   <body>
-
-  <div class="mdl-layout mdl-js-layout mdl-layout--fixed-header">
-  <header class="mdl-layout__header">
-    <div class="mdl-layout__header-row">
-      <!-- Title -->
-      <span class="mdl-layout-title">Node.js Core Coverage</span>
-      <!-- Add spacer, to align navigation to the right -->
-      <div class="mdl-layout-spacer"></div>
-      <!-- Navigation. We hide it in small screens. -->
-      <nav class="mdl-navigation mdl-layout--large-screen-only">
-        <a class="mdl-navigation__link" href="https://github.com/nodejs/node">Node.js Core</a>
-        <a class="mdl-navigation__link" href="https://github.com/addaleax/node-core-coverage">Project</a>
-      </nav>
-    </div>
+  <header>
+     <div class="container" id="logo">
+         <img src="https://nodejs.org/static/images/logos/nodejs-new-white-pantone.png" alt="node.js">
+     </div>
   </header>
+
+
+  <div id="main">
+      <div class="container">
+
+          <h1>Node.js Code Coverage</h1>
+
   <div class="mdl-layout__drawer">
     <nav class="mdl-navigation">
         <a class="mdl-navigation__link" href="https://github.com/nodejs/node">Node.js Core</a>
```

Comment:
I think the line after this one (link to `https://github.com/addaleax/node-core-coverage`) can be removed